packages/ckit/src/providers/mercury/MercuryProvider.ts

```diff
@@ -227,6 +227,46 @@ export class MercuryProvider extends AbstractProvider {
     return acc.cells;
   }
 
+  async collectUdtCellsByMinCkb(
+    address: Address,
+    udt: CkbTypeScript,
+    minimalCapacity: HexNumber,
+  ): Promise<ResolvedOutpoint[]> {
+    const lock = this.parseToScript(address);
+    const searchKey: SearchKey = {
+      script: lock,
+      script_type: 'lock',
+      filter: {
+        script: udt,
+        output_capacity_range: [minimalCapacity, '0xffffffffffffffff'],
```

Comment:
maybe it is unnecessary here, we can collect multiple cells and merge them into one like this

```yaml
# collected cells
cell1:
  lock: Alice
  type: sUDT
  capacity: capacity1
  amount: amount1
cell2: 
  lock: Alice
  type: sudt
  capacity: capacity2
  amount: amount2
```

```yaml
# merge
inputs:
  - cell1
  - cell2
outputs:
  - cell'
    lock: Alice
    type: sUDT
    amount: amount1 + amount2 - transferedAmount
    capacity: capacity1 + capacity2 - transferedCapacity
```

Reply:
I have merge them into one cell.